adapters/connatix/connatix.go

```diff
@@ -222,15 +222,20 @@ func buildRequestImp(imp *openrtb2.Imp, ext impExtIncoming, displayManagerVer st
 		imp.BidFloor = convertedValue
 	}
 
-	impExt := impExt{
```

Comment:
With this change, `impExt` defined in `adapters/connatix/models.go` is no longer used. Should we remove it?
```
11   type impExtIncoming struct {
12       Bidder openrtb_ext.ExtImpConnatix `json:"bidder"`
13   }
14   
15 - type impExt struct {
16 -     Connatix impExtConnatix `json:"connatix"`
17 - }
18   
19   type impExtConnatix struct {
adapters/connatix/models.go
```

Reply:
@guscarreon thank you! addressed